src/monodroid/jni/debug-app-helper.cc

```diff
@@ -83,6 +81,20 @@ Java_mono_android_DebugRuntime_init (JNIEnv *env, jclass klass, jobjectArray run
 	androidSystem.set_override_dir (0, androidSystem.get_primary_override_dir ());
 	androidSystem.setup_app_library_directories (env, runtimeApks, applicationDirs, androidApiLevel);
 
+	jstring_wrapper jstr (env);
+	jstr = env->GetObjectArrayElement (externalStorageDirs, 0);
+	androidSystem.set_override_dir (1, utils.strdup_new (jstr.get_cstr ()));
+
+	jstr = env->GetObjectArrayElement (externalStorageDirs, 1);
+	androidSystem.set_override_dir (2, utils.strdup_new (jstr.get_cstr ()));
+
+	for (uint32_t i = 0; i < BasicAndroidSystem::MAX_OVERRIDES; ++i) {
+		const char *p = androidSystem.get_override_dir (i);
+		if (!utils.directory_exists (p))
```

Comment:
This is slightly concerning, in that it's *imaginable* that `externalStorageDirs` may be null at some point -- especially when we're looking into shared runtime removal/etc. -- in which case it's *imaginable* that `p` will be `nullptr` here, which would (eventually) result in `stat(nullptr, &buf)`, which is....probably "fine"?  But not desirable?